### ISSUE TITLE
Sequence annotation front end display error

### ISSUE THREAD

**Issue by oldpiao** [doccano/doccano#2025]
在python3.8版本上部署，doccano版本为1.8.0
问题：有时会发现在前端标注过的标签未显示出来。一般出现在当前页面标签较多，或同一段文本标注多个标签时。一般情况使用F5刷新一下就可以解决，但有时刷新后依然未显示出来。

Deployed on python3.8, doccano is 1.8.0

Problem: Sometimes you will find that the labels annotated in the front end are not displayed. This parameter is usually displayed when there are many tags on the current page or when multiple tags are labeled in the same text. Generally, F5 can be used to refresh the problem, but sometimes it still does not show up.

F5 to refresh before
![微信图片_20221020191847](https://user-images.githubusercontent.com/30197596/196934926-62443283-de3e-442e-bbed-7d88d4f43369.jpg)

After the F5 to refresh
![image](https://user-images.githubusercontent.com/30197596/196935086-ec0d70b5-abd6-4b71-b075-d188eed48b26.png)

**Comment by VulnDetector** [comment]
Yeah,I have the same situation when using sequence labeling with NER and relation.

**Comment by lance132** [comment]
Hi, we're also encountering this problem. Some digging through older issues has revealed this one #1596 and this one #1880 which all seem to be referring to the same bug.
